core/templates/dev/head/pages/profile-page/profile-page.directive.html

```diff
@@ -117,15 +117,25 @@ <h2 class="oppia-profile-username-small-screen">
     <span class="oppia-profile-portfolio-pages" ng-if="$ctrl.numUserPortfolioExplorations > 6">
       <i class="material-icons md-18 oppia-navigation-button"
          ng-disabled="$ctrl.startingExplorationNumber === 1"
-         ng-click="$ctrl.goToPreviousPage()">
+         ng-click="$ctrl.goToPreviousPage()"
+         ng-class="{'oppia-navigation-mouse-pointer': $ctrl.startingExplorationNumber !== 1}">
         &#xE5C4;
       </i>
       Showing <[$ctrl.startingExplorationNumber]> - <[$ctrl.endingExplorationNumber]> of <[$ctrl.numUserPortfolioExplorations]>
       <i class="material-icons md-18 oppia-navigation-button"
          ng-disabled="$ctrl.endingExplorationNumber >= $ctrl.numUserPortfolioExplorations"
-         ng-click="$ctrl.goToNextPage()">
+         ng-click="$ctrl.goToNextPage()"
+         ng-class="{'oppia-navigation-mouse-pointer': $ctrl.endingExplorationNumber < $ctrl.numUserPortfolioExplorations}">
         &#xE5C8;
       </i>
     </span>
   </md-card>
 </div>
+<style>
```

Comment:
You can leave it here, but do indent like this file: https://github.com/oppia/oppia/blob/ba55ea9b5df8d6c8d3d2b82ca0c11800f8409d81/extensions/interactions/ImageClickInput/directives/image-click-input-interaction.directive.html

Reply:
I am sorry, I didn't notice my mistake in the indentation. I have now corrected it.